contracts/utils/convex/ConvexYieldWrapper.sol

```diff
@@ -91,12 +97,11 @@ contract ConvexYieldWrapper is ConvexStakingWrapper {
         //add up all balances of all vaults
         uint256 collateral;
         DataTypes.Balances memory balance;
-        for (uint256 i = 0; i < userVault.length; i++) {
-            if (userVault[i] != bytes12(0)) {
```

Comment:
if we revert the array cleanup in removeVault then we should add this back

Reply:
I like the cleanup